dataikuapi/dss/projectfolder.py

```diff
@@ -0,0 +1,236 @@
+        """
+        params = {
+            "destination": destination.project_folder_id
+        }
+        self.client._perform_empty("POST", "/project-folders/%s/move" % self.project_folder_id, params=params)
+
+    ########################################################
+    # Project move
+    ########################################################
+    def move_project_to(self, project_key, destination):
+        """
+        Move a project within this project folder into another project folder (aka. destination)
+
+        :param str project_key: the project key associated with the project to move
+        :param destination: the project folder to put this project into
+        :type destination: A :class:`dataikuapi.dss.projectfolders.DSSProjectFolder`
+        """
+        params = {
+            "destination": destination.project_folder_id
+        }
+        self.client._perform_empty("POST", "/project-folders/%s/projects/%s/move" % (self.project_folder_id, project_key), params=params)
+
+class DSSProjectFolderSettings(object):
+    """
+    A handle to interact with project folder settings
+
+    Do not create this class directly, instead use :meth:`dataikuapi.dss.projectfolder.DSSProjectFolder.get_settings`
+    """
+    def __init__(self, client, project_folder_id, settings):
+        self.client = client
+        self.project_folder_id = project_folder_id
+        self.settings = settings
+
+    def get_raw(self):
+        """Gets all settings as a raw dictionary. This returns a reference to the raw settings, not a copy,
+        so changes made to the returned object will be reflected when saving.
+
+        :rtype: dict
+        """
+        return self.settings
+
+    def get_name(self):
+        """Get the name of the project folder
+
+        :returns str: the current name of the project folder
+        """
+        return self.settings.get("name", None)
+
+    def set_name(self, name):
+        """Set the name of the project folder
+
+        :param str name: the new name of the project folder
+        """
+        self.settings["name"] = name
+
+    def get_owner(self):
+        """Get the login of the owner of the project folder
+
+        :returns str: the current login owner of the project folder
+        """
+        return self.settings.get("owner", None)
+
+    def set_owner(self, owner):
+        """Set the owner of the project folder
+
+        :param str owner: the new owner login of the project folder
+        """
+        self.settings["owner"] = owner
+
+    def get_permissions(self):
+        """Get the permissions of the project folder
+        Warning: the returned list is the direct object of the settings, modifying it will modify the current settings
+
+        :return list: the current permissions of the project folder
+        """
+        return self.settings["permissions"]
+
+    def set_permissions(self, permissions):
```

Comment:
Since we return directly, do we really need a set_ call ? I don't think we do it for other similar cases